packages/apps/shopify-app-react-router/src/server/__test-helpers/setup-embedded-flow.ts

```diff
@@ -9,42 +7,11 @@ import {testConfig} from './test-config';
 
 export async function setUpEmbeddedFlow() {
   const shopify = shopifyApp({
-    ...testConfig({
-      restResources,
-    }),
-    future: {
-      removeRest: false,
-      unstable_newEmbeddedAuthStrategy: false,
```

Comment:
The diff for this file seems weird. I left `unstable_newEmbeddedAuthStrategy: false,`
https://github.com/Shopify/shopify-app-js/blob/e019f59b4a1071902662b1f6d9f8649b5a3e79b2/packages/apps/shopify-app-react-router/src/server/__test-helpers/setup-embedded-flow.ts